### ISSUE TITLE
Custom theme/extend/screens with "/" in name not working, e.g. "ar-3/4"

### ISSUE THREAD

**Issue by fightingdreamer** [tailwindlabs/tailwindcss#10327]
**What version of Tailwind CSS are you using?**
3.2.4
**What build tool (or framework if it abstracts the build tool) are you using?**
postcss: 8.4.20
**What version of Node.js are you using?**
18.12.1
**What browser are you using?**
Safari (latest)
Chrome (latest)
**What operating system are you using?**
MacOS
**Reproduction URL**
https://play.tailwindcss.com/gMqETne4se
**Describe your issue**
When I create custom @media query rule using theme/extend/screens and include "/" in class name, that rule is not working.
For reference I swapped "/" with "_" in reproduction URL to show that this rule is working correctly.

I need "/" to create rules for aspect ratio, e.g. "ar-16/9".

**Comment by thecrypticace** [comment]
Hey! I've merged the fix for this in #10336. It'll be available in our next release. In the meantime you can test it using our insiders build: `npm install tailwindcss@insiders`

**Comment by fightingdreamer** [comment]
Wow, that was super fast response!

I tried to install it, version `tailwindcss@0.0.0-insiders.91b4b93`, but got error about missing dependency `oxide-node-api-shim`. I will test again after it will be pushed to stable, unless I find a way install it.

Error details:
```
> pnpm add 'tailwindcss@insiders'
ERR_PNPM_LINKED_PKG_DIR_NOT_FOUND  Could not install from "/PATH_TO_MY_PROJECT/oxide-node-api-shim" as it does not exist.
```
